src/core_plugins/tile_map/public/base_maps_visualization.js

```diff
@@ -145,10 +145,8 @@ export function BaseMapsVisualizationProvider(serviceSettings, i18n) {
           const firstRoadMapLayer = tmsServices.find((s) => {
             return s.id === 'road_map';//first road map layer
           });
-          const fallback = firstRoadMapLayer ? firstRoadMapLayer : tmsServices[0];
-          if (fallback) {
-            this._setTmsLayer(firstRoadMapLayer);
-          }
+          const initMapLayer = firstRoadMapLayer ? firstRoadMapLayer : tmsServices[0];
```

Comment:
it is possible that `fallback` does not exist. it will occur when EMS-connection is turned off and there is no tms-service.

Reply:
The next line accounts for this. ` if (initBasemapLayer) { this._setTmsLayer(initBasemapLayer); }` So if no basemap was sent through, there will be no attempt to set it.